integration/shareddata/scalars.go

```diff
@@ -245,26 +243,6 @@ var DateTimes = &Values[string]{
 	},
 }
 
-// Nulls contains null value for tests.
-var Nulls = &Values[string]{
-	name:     "Nulls",
-	handlers: []string{"pg", "tigris"},
-	validators: map[string]map[string]any{
-		"tigris": {
-			"$tigrisSchemaString": `{
-				"title": "%%collection%%",
-				"primary_key": ["_id"],
-				"properties": {
-					"_id": {"type": "string"}
-				}
-			}`,
-		},
-	},
-	data: map[string]any{
-		"null": nil,
```

Comment:
This test case is not possible anymore because we were trying to set `null` in the field `v`, which was not described in the schema. In beta.27 it's not possible anymore to set such a field.